test/1_test_fullpath.js

```diff
@@ -84,14 +87,881 @@ contract('Voucher tests', async (addresses) => {
+            ev._nonce.eq(new BN(2)) &&
+            ev._seller === users.seller.address &&
+            ev._validFrom.eq(new BN(constants.PROMISE_VALID_FROM)) &&
+            ev._validTo.eq(new BN(constants.PROMISE_VALID_TO)) &&
+            ev._idx.eq(new BN(1))
+          );
+        },
+        'promise event incorrect'
+      );
+
+      const internalTokenTx = await truffleAssert.createTransactionResult(
+        contractERC1155ERC721,
+        txOrder2.tx
+      );
+
+      truffleAssert.eventEmitted(
+        internalTokenTx,
+        'TransferSingle',
+        (ev) => {
+          return (
+            ev._operator === contractVoucherKernel.address &&
+            ev._from === constants.ZERO_ADDRESS &&
+            ev._to === users.seller.address &&
+            ev._id.eq(tokenSupplyKey2) &&
+            ev._value.eq(new BN(constants.ORDER_QUANTITY2))
+          );
+        },
+        'transfer event incorrect'
+      );
+
+      //Check BosonRouter state
+      assert.equal(
+        await contractBosonRouter.correlationIds(users.seller.address),
+        2,
+        'Correlation Id incorrect'
+      );
+
+      //Check VocherKernel State
+      const promise = await contractVoucherKernel.promises(promiseId2);
+      assert.strictEqual(promise.promiseId, promiseId2, 'Promise Id incorrect');
+      assert.isTrue(promise.nonce.eq(new BN(2)));
+      assert.strictEqual(
+        promise.seller,
+        users.seller.address,
+        'Seller incorrect'
+      );
+      assert.isTrue(promise.validFrom.eq(new BN(constants.PROMISE_VALID_FROM)));
+      assert.isTrue(promise.validTo.eq(new BN(constants.PROMISE_VALID_TO)));
+      assert.isTrue(promise.price.eq(new BN(constants.PROMISE_PRICE2)));
+      assert.isTrue(promise.depositSe.eq(new BN(constants.PROMISE_DEPOSITSE2)));
+      assert.isTrue(promise.depositBu.eq(new BN(constants.PROMISE_DEPOSITBU2)));
+      assert.isTrue(promise.idx.eq(new BN(1)));
+
+      const orderPromiseId = await contractVoucherKernel.ordersPromise(
+        tokenSupplyKey2
+      );
+      assert.strictEqual(
+        orderPromiseId,
+        promiseId2,
+        'Order Promise Id incorrect'
+      );
+
+      const tokenNonce = await contractVoucherKernel.tokenNonces(
+        users.seller.address
+      );
+      assert.isTrue(tokenNonce.eq(new BN(2)));
+
+      //Check ERC1155ERC721 state
+      const sellerERC1155ERC721BalanceVoucherSet1 = await contractERC1155ERC721.balanceOf(
+        users.seller.address,
+        tokenSupplyKey1
+      );
+      assert.isTrue(sellerERC1155ERC721BalanceVoucherSet1.eq(new BN(1)));
+
+      const sellerERC1155ERC721BalanceVoucherSet2 = await contractERC1155ERC721.balanceOf(
+        users.seller.address,
+        tokenSupplyKey2
+      );
+      assert.isTrue(sellerERC1155ERC721BalanceVoucherSet2.eq(new BN(1)));
+    });
+  });
+
+  describe('Commit to buy a voucher (ERC1155)', () => {
+    beforeEach('setup contracts for tests', async () => {
```

Comment:
Do we setup the contracts here?

Reply:
Changed to "perform prerequisite steps". Will be included in next commit.